specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2339,7 +2339,7 @@
         "Utf16CodeUnit"
       ],
       "x-ms-enum": {
-        "name": "StringIndexTypeResponse",
+        "name": "StringIndexType",
```

Comment:
This creates 2 types in .NET `StringIndexType` and `StringIndexTypeResponse`, where the first is used for sync endpoints and later in async (analyze) endpoints.
I tested this change in .NET and it generates everything correctly.